### PR TITLE
[ES] Add spanish transalation of comprehensions lesson

### PR DESCRIPTION
**Pull request by erickgnavar**

### CODE REVIEW

**Comment by doomspork** [comment]
@erickgnavar if @milmazz isn't able to review these we can go ahead and merge.  I will do a better job of trying to recruit additional translators to help 😁

**Comment by milmazz** [comment]
@doomspork @erickgnavar Hey guys, I can take a look in a few hours.

File: es/lessons/basics/comprehensions.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+4 - 2
+4 - 3
+4 - 4
+```
+
+Las comprensiones de listas son azúcar sintáctica y solo deberían ser usadas cuando es apropiado.
+
+## Filtros
+
+Puedes pensar en los filtros como un tipo de guardas para las comprensiones. Cuando un valor filtrado retorna `false` o `nil` este es ejecutado desde el final de la lista. Vamos a iterar un rango y solo preocuparnos por los números pares. Vamos a usar la función `is_even/1` del módulo `Integer` para verificar si el valor es par o no.
```
Comment (milmazz) [inline es/lessons/basics/comprehensions.md]: _Cuando un valor filtrado retorna `false` o `nil` este es **excluido** de la lista final._

Comment (erickgnavar) [reply es/lessons/basics/comprehensions.md]: Fixed

**Pull request merged**